lms/templates/ccx/grading_policy.html

```diff
@@ -3,7 +3,7 @@
 <div id="warn-coach" class="wrapper-msg urgency-high warning">
   <div class="msg">
     <i class="msg-icon icon-warning-sign"></i>
-    <div class="msg-content">
+    <div class="msg-content" tabindex="0">
```

Comment:
@amir-qayyum-arbisoft I still do not know why this is being added to the tab order.  If a screen reader user can tab to something, they expect that it is something they can interact with.  This is just a non-interactive div.  This content is already perfectly accessible without this change.

Sometimes people who test with screen readers think that if you can't tab to something then its inaccessible.  This is not true.  Screen reader users use many methods to interact with a page.  In Voiceover and Chromevox, users use their screen reader key + arrows to move focus to all blocks of content.  On JAWS, NVDA and other screen readers, users use the arrow keys to move through all of the content on the page.  They would encounter and consume this just fine.  Forcing it into the tab order is unnecessary and will cause confusion.